infrastructure/create-github-environment.js

```diff
@@ -25,11 +35,13 @@ const config = {
     DOMAIN: '', // web domain applied after all public subdomains
     REPLICAS: '1'
   },
+  sms: {
+    INFOBIP_API_KEY: process.env.INFOBIP_API_KEY,
```

Comment:
Should add `| ''` if not supplied

Reply:
I added a check in the script later on to verify none of the variables are undefined or empty strings. I did this after I noticed Github failing the requests if an empty string is passed as the value of a secret. 

Now you have to explicitly do `--sms-enabled` if you want to configure an SMS gateway and only then will this block of variables be applied to the secrets. This could be made a question prompt in the script if the parameter isn't supplied